### ISSUE TITLE
Abnormal vestibulo-spinal reflexes

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10638]
**Preferred term label:**

**Synonyms**
Impaired vestibulospinal reflexes

**Definition (free text, please give PubMed ID)**
Manzoni, D. (2009). Vestibulo-spinal Reflexes. In: Binder, M.D., Hirokawa, N., Windhorst, U. (eds) Encyclopedia of Neuroscience. Springer, Berlin, Heidelberg. https://doi.org/10.1007/978-3-540-29678-2_6315 : The vestibulospinal (VS) reflexes are changes in the activity of body muscles induced by movements of the head in space that stimulate and aimed at stabilizing.
Characteristics : Functions
The VS reflexes contribute to the control of postural orientation. In this task, they continuously collaborate with reflexes elicited by stimulation of visual and somatosensory receptors. Labyrinthine receptors are sensitive to modifications (i) of the head position with respect to the earth vertical or (ii) of the angular/linear velocity of head motion in space. This information can be used to monitor deviations from the maintained body position.

PMID: 29081426 These symptoms in BVP are caused by the sensory vestibular deficit leading to **impaired vestibulo-spinal reflexes** leading to higher body sway and broad-based gait.

PMID: 38178884 Two of the main physical symptoms of BVP are movement induced blurred vision (oscillopsia) and unsteadiness when walking or standing which often worsens on uneven ground or in darkness. These symptoms are primarily due to impaired vestibular-ocular and **vestibular-spinal reflexes**.

**Parent term (use hpo.jax.org/app)**
Abnormal vestibular function HP:0001751

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Idiopathic bilateral vestibulopathy ORPHA:171684

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001104
- label: Abnormal vestibulo-spinal reflex
- parents: [Abnormal vestibular function (HP:0001751)](https://hpo.jax.org/app/browse/term/HP:0001751)
- pmid: PMID:29081426|PMID:38178884
- def: An abnormality of the vestibulospinal (VS) reflexes, which contribute to the control of postural orientation. In this task, they continuously collaborate with reflexes elicited by stimulation of visual and somatosensory receptors. Labyrinthine receptors are sensitive to modifications (i) of the head position with respect to the earth vertical or (ii) of the angular/linear velocity of head motion in space. This information can be used to monitor deviations from the maintained body position.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10638
- synonyms:
